src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
Thanks. It looks like GitHub search is broken because it doesn't return `System.Runtime.cs` even though the change is in master: https://github.com/dotnet/corefx/search?q=SchemeDelimiter